ydb/mvp/oidc_proxy/oidc_session_create_nebius.h

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include "oidc_session_create.h"
+
+namespace NMVP {
+
+class THandlerSessionCreateNebius : public THandlerSessionCreate {
+private:
+    using TBase = THandlerSessionCreate;
+
+public:
+    THandlerSessionCreateNebius(const NActors::TActorId& sender,
+                                const NHttp::THttpIncomingRequestPtr& request,
+                                const NActors::TActorId& httpProxyId,
+                                const TOpenIdConnectSettings& settings)
+        : THandlerSessionCreate(sender, request, httpProxyId, settings)
+        {}
+
+    void RemoveAppliedCookie(const TString& cookieName) override {
+        ResponseHeaders.Set("Set-Cookie", TStringBuilder() << cookieName << "=; Path=" << GetAuthCallbackUrl() << "; Max-Age=0");
+        ResponseHeaders.Set("Set-Cookie", TStringBuilder() << CreateNameSessionCookie(Settings.ClientId) << "=; Max-Age=0");
```

Comment:
Is it valid to set two headers with the same name?
We have TMap for headers here, not multimap((